src/Ensconce.Update/TagDictionary.cs

```diff
@@ -32,7 +32,7 @@ public static TagDictionary Empty()
         public static TagDictionary FromIdentifier(string identifier)
         {
             var tagDictionary = new TagDictionary();
-            tagDictionary.LoadDictionary(identifier, new Dictionary<TagSource, string>());
+            tagDictionary.LoadDictionary(identifier, new Dictionary<TagSource, string> { { TagSource.Environment, "" } });
             return tagDictionary;
```

Comment:
unclear what this change is for?

Reply:
Is this the change to reinstate the reading (and output) of files as UTF8 @BlythMeister  ?

Reply:
no, this change is so that when you don't have a fixed structure you still get the environment variables in your tag dictionary

Reply:
gotcha, ta